### PR TITLE
Wrong currency sign on creditmemo_grid & sales_order_view > invoice grid

### PR DESCRIPTION
**Pull request by latenights**
### Description (*)

Added missing column `base_currency_code` & `order_currency_code` to table `sales_creditmemo_grid`. Price column rendering changed on `sales_creditmemo_index` (Creditmemo grid) & `sales_order_view` - Invoice tab.

### Fixed Issues
1. magento/magento2#22490

### Manual testing scenarios (*)

1. Create order in different currency as configured on default store view
2. Go to Backend > SALES > Credit Memos, create Credit Memo
3. Check Refunded & Subtotal column on Credit Memo grid (`sales_creditmemo_index`). You should see price in currency configured on default store view (base_currency_code).
4. Go to Backend > SALES > Orders > choose your order and check _Invoice grid_ in Invoices tab.  **Amount** should be `base grand total` with `base_currency_code`.

sales_order_view > Credit Memos (base grand total):
image

sales_order_view > Invoices (grand total):
image

sales_creditmemo_index (wrong currency sign):
image

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by orlangur** [comment]
Hey @latenights, could you please

> I mean please report an issue and just link it in both PRs so that this bug and its fix may be properly tracked.

**Comment by latenights** [comment]
@orlangur Issue https://github.com/magento/magento2/issues/22490 created.

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5194](https://jira.corp.magento.com/browse/ENGCOM-5194) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@latenights thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed 

**Before:**

before

**After:**

after

**Comment by magento-engcom-team** [comment]
@latenights thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-engcom-team** [comment]
@latenights thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-engcom-team** [comment]
@latenights thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: app/code/Magento/Sales/Setup/Patch/Data/UpdateCreditmemoGridCurrencyCode.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+     * @inheritdoc
+     */
+    public function apply()
+    {
+        $salesSetup = $this->salesSetupFactory->create(['setup' => $this->moduleDataSetup]);
+        $connection = $salesSetup->getConnection();
+        $creditMemoGridTable = $salesSetup->getTable('sales_creditmemo_grid');
+        $orderTable = $salesSetup->getTable('sales_order');
+        // phpcs:disable Magento2.SQL.RawQuery
+        $sql = "UPDATE {$creditMemoGridTable} AS scg
```
Comment (sidolov) [inline app/code/Magento/Sales/Setup/Patch/Data/UpdateCreditmemoGridCurrencyCode.php]: It's not a good idea to left raw SQL here, @Nazar65 please refactor with adaptor usage.

Comment (ysapiga) [reply app/code/Magento/Sales/Setup/Patch/Data/UpdateCreditmemoGridCurrencyCode.php]: @sidolov please review changes

**Comment by magento-engcom-team** [comment]
@latenights thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Pull request merged**